doc/source/wsrep-system-index.rst

```diff
@@ -375,6 +375,57 @@ Possible values for this variable are:
 .. |binlog_format| replace:: ``binlog_format``
 .. _binlog_format: https://dev.mysql.com/doc/refman/5.7/en/binary-log-setting.html
 
+.. variable:: wsrep_min_log_verbosity
+
+   :cli: ``--wsrep-min-log-verbosity``
+   :conf: Yes
+   :scope: Global
+   :dyn: Yes
+   :default: 3
+
+This variable defines the *minimum* logging verbosity for the given
+|pxc| server and has the same values as the ``log_error_verbosity``
+variable.
+
+The effective log verbosity cannot be lower than the value of
```

Comment:
As above. New variable affects only wsrep/galera part.

It defines minimum desired verbosity from wsrep/galera regardless of what is set to log_error_verbosity'.

Example:
wsrep_min_log_verbosity = 3;
log_error_verbosity = 1;
We see only error/system logs from server parts + all logs from wsrep/galera part

wsrep_min_log_verbosity = 1;
log_error_verbosity = 3;
We see all logs from server part and all logs from wsrep/galera part

Please refer to the table included in the Jira ticket.